src/pawns.cpp

```diff
@@ -172,6 +175,13 @@ namespace {
             backward = (b | shift_bb<Up>(b)) & theirPawns;
         }
 
+        if (!backward && relative_rank(Us, s) == RANK_5)
+        {
+            if (pawn_attack_span(Us, s + pawn_push(Us)) & theirPawns)
+                score += EnPassantBonus;
+
+        }
+
```

Comment:
Good point, it could all be put on one line. I wrote it that way mostly because it looked clearer, but it's possible that the compiler won't optimise it to identical code.

Presumably that simplification won't require a re-test?